### DISCUSSION TITLE
Akuvox Intercom Integration

### DISCUSSION THREAD

**Discussion by robotsmarthomes** [koush/scrypted#1565] [General]
Hello,

I was hoping for support for Akuvox Particulary model E16C.

Is this something that is possible?

Thanks

**Comment by mgoeller** [comment]
Did you try and succeed? Also with the doorbell button?

**Comment by robotsmarthomes** [comment]
Hey,

We gave up on Akuvox.

It’s just not compatible. Not with ONVIF or any other integration.

Very unfortunate
